CTP/doc/sql/sql_guide.md

```diff
@@ -0,0 +1,524 @@
+# 1 Test Objective
+SQL is an ANSI (American National Standards Institute) standard computer language, but there are still many different versions of SQL.However, in order to be compatible with ANSI standards, they must all support some major commands (such as SELECT, UPDATE, DELETE, INSERT, WHERE, and so on) in a similar way.  
```

Comment:
1. Look strange for expression for the first statement.
2. "CUBRID SQL detection" ??

Please refine expression.

Reply:
Please improve by suggestion.